composer.json

```
@@ -25,7 +25,8 @@
     },
     "require-dev": {
         "doctrine/doctrine-bundle": "^1.6.12|^2.0",
-        "doctrine/dbal": "~2.2",
+        "doctrine/dbal": "^2.13.1|^3.1",
```

If it's just about `executeStatement()`, we can relax the constraint a little:

```suggestion
        "doctrine/dbal": "^2.13|^3.0",
```

---

I just copy/pasted the constraint and conflict from the component.  Especially since the component effectively isn’t going to allow the couple of patch releases this change would enable.

---

Yes, but the bundle is able to load earlier versions of the component. But I think we can leave your constraint as it is now.